timestep.c

```diff
@@ -36,9 +37,7 @@ void set_global_time(double newtime) {
 #ifdef LIGHTCONE
     lightcone_set_time(All.cf.a);
 #endif
-#ifdef COOL
```

Comment:
This is it. This is it. I think this is why the star formation was all wrong. IonizeParams are always set to z=0 if this line is not enabled -- which means additional heating and less star formation..